code/game/objects/effects/lobby_art.dm

```diff
@@ -0,0 +1,46 @@
+GLOBAL_LIST_EMPTY(lobby_art_files)
+
+/obj/effect/lobby_art
```

Comment:
?? why

Reply:
Why what? `effect` types are good for abstract things that shouldn't be interacted with.

Reply:
It was a turf earlier though?

Reply:
And now it's not.